### DISCUSSION TITLE
How long does a query stay cached?

### DISCUSSION THREAD

**Discussion by Vasques1995** [zino-app/graphql-flutter#1001] [Q&A] [answered]
How long does a standard query on 5.0.0 stay cached and can this value be controlled trough the GraphQL Server response?

**Comment by vasilich6107** [comment, marked as answer]
forever)
There is no cache invlidation mechanisms built in
The default behaviour is to run query -> store data in cache -> on query re-run take data from cache -> execute real query behind the scene -> update cache with new data -> update UI

But the real behaviour depends on `fetchPolicy`
